.github/workflows/pr.yml

```diff
@@ -33,14 +38,15 @@ jobs:
         with:
           path: 'repo'
       - name: KICS Github Action
-        uses: checkmarx/kics-action@v1.4
+        uses: checkmarx/kics-action@v1.5
         with:
           path: 'repo'
           exclude_paths: 'repo/examples'
           output_path: 'results'
           platform_type: terraform
           fail_on: high,medium
-          exclude_queries: '8e94dced-9bcc-4203-8eb7-7e41202b2505,e592a0c5-5bdb-414c-9066-5dba7cdea370,7ebc9038-0bde-479a-acc4-6ed7b6758899,91f16d09-689e-4926-aca7-155157f634ed,a88baa34-e2ad-44ea-ad6f-8cac87bc7c71' # Excludes 'Auto Scaling Group With No Associated ELB', 'IAM Access Analyzer Undefined' queries, 'Vulnerable policy for the KMS Key' ECS Service without tasks running and excludes where password equals null.
```

Comment:
I like that you've moved these to envs, but could you add the comments to each env var as they are the vulnerability titles

Reply:
Yep sure. Added a comment with their query titles next to each env var.